app/assets/stylesheets/costs/costs_legacy.css

```diff
@@ -97,6 +97,10 @@ td.comment input {
   cursor: pointer;
 }
 
+#cost_object_fixed_date {
+  width: 10%;
+}
+
```

Comment:
@HDinger you can simply apply the `-xslim` modifier class on the field. That way we do not introduce another css rule.